tests/test_pysnooper.py

```diff
@@ -1172,3 +1173,21 @@ def sum_to_x(x):
             ReturnValueEntry('49995000'),
         )
     )
+
+
+def test_activate_deactivate_snoop():
+    string_io = io.StringIO()
+
+    def my_function(foo):
+        x = 7
+        y = 8
+        return x + y
+
+    os.environ['PYSNOOPER_DISABLED'] = '1'
```

Comment:
This won't work because DISABLED is a constant. You need something like:

```suggestion
    pysnooper.tracer.DISABLED = '1'
```